packages/browserify/test/fixtures/secureBundling/lavamoat/node/policy.json

```diff
@@ -85,6 +85,7 @@
       "globals": {
         "__dirname": true,
         "__filename.slice": true,
+        "console.warn": true,
```

Comment:
console.warn is there, should have been listed before. Anyway, this change is not undermining the test.